### PR TITLE
Feature/ocpi update

### PR DESCRIPTION
**Pull request by dv-rosales**

### CODE REVIEW

File: src/main/kotlin/snc/openchargingnetwork/node/models/ocpi/Commands.kt
Comment (matheusrosendo) [inline src/main/kotlin/snc/openchargingnetwork/node/models/ocpi/Commands.kt:1]: great, I already tested it. Worked fine!

File: src/test/kotlin/snc/openchargingnetwork/node/controllers/ocpi/v2_2/CredentialsControllerTest.kt
Comment (matheusrosendo) [inline src/test/kotlin/snc/openchargingnetwork/node/controllers/ocpi/v2_2/CredentialsControllerTest.kt:1]: I never programmed in kotlin, but doesnt it allow tokenB to be null?

**Review by matheusrosendo — APPROVED** [review]
Only a concern about tokenB being null, looks good overall!

File: src/test/kotlin/snc/openchargingnetwork/node/controllers/ocpi/v2_2/CredentialsControllerTest.kt
Comment (dv-rosales) [reply src/test/kotlin/snc/openchargingnetwork/node/controllers/ocpi/v2_2/CredentialsControllerTest.kt:1]: TokenB is null until the party calls the POST credentials endpoint, after that it shouldn't  be null

**Pull request merged**